recipe/meta.yaml

```
@@ -1,34 +1,40 @@
 {% set name = "kopf" %}
-{% set version = "1.36.0" %}
+{% set version = "1.38.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b618c0cd19398681dd4a0a263a0a265b5267a05064a5d0c69025d0ed710bd0af
+  sha256: 7b9f83240269636e00e4b11e9c7c8afbfefba3d3326ac2d2aa7904d35c4f89cd
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```

```suggestion
  number: 0
  entry_points:
    - kopf=kopf.cli:main
```
see https://github.com/nolar/kopf/blob/1.38.0/setup.py#L52C14-L52C34